golang/main.go

```diff
@@ -2,57 +2,33 @@ package main
 
 import (
 	"fmt"
-	"strconv"
-
-	"github.com/astaxie/beego"
+	"io/ioutil"
+	"net/http"
+	"os"
 )
 
-// The main function defines a single route, its handler
-// and starts listening on port 8080 (default port for Beego)
-func main() {
-	/* This would match routes like the following:
-	   /sum/3/5
-	   /product/6/23
-	   ...
-	*/
-	fmt.Println("Starting beego server...")
-	beego.Router("/:operation/:num1:int/:num2:int", &mainController{})
-	beego.Run()
-}
+const namespaceFile = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
 
-// This is the controller that this application uses
-type mainController struct {
-	beego.Controller
+func main() {
+	fmt.Println("Starting hello-world server...")
+	http.HandleFunc("/", helloServer)
+	http.ListenAndServe(":8080", nil)
 }
 
-// Get() handles all requests to the route defined above
-func (c *mainController) Get() {
-	//Obtain the values of the route parameters defined in the route above
-	operation := c.Ctx.Input.Param(":operation")
-	num1, _ := strconv.Atoi(c.Ctx.Input.Param(":num1"))
-	num2, _ := strconv.Atoi(c.Ctx.Input.Param(":num2"))
-
-	//Set the values for use in the template
-	c.Data["operation"] = operation
-	c.Data["num1"] = num1
-	c.Data["num2"] = num2
-	c.TplName = "result.html"
-
-	// Perform the calculation depending on the 'operation' route parameter
-	switch operation {
-	case "add":
-		c.Data["result"] = add(num1, num2)
-	case "mult":
-		c.Data["result"] = multiply(num1, num2)
-	default:
-		c.TplName = "invalid-route.html"
-	}
+func helloServer(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprint(w, helloMessage())
 }
 
-func add(n1, n2 int) int {
-	return n1 + n2
+func helloMessage() string {
+	namespace := getNamespace()
+	return fmt.Sprintf("Hello from the cluster namespace %s", namespace)
 }
 
-func multiply(n1, n2 int) int {
-	return n1 * n2
+func getNamespace() string {
+	bytes, err := ioutil.ReadFile(namespaceFile)
+	if err != nil {
+		fmt.Println("Error reading current namespace")
```

Comment:
I'd add an error that makes it clear that this needs to be run inside the cluster. And include the `err` in the message?